hugo/layouts/papers/single.html

```diff
@@ -70,6 +114,182 @@
           $(".btn-clipboard-outside").removeClass("d-none");
       }
   });
+
+  const paper_params = {
+    anthology_id: "{{ .Params.anthology_id }}",
+    title: "{{ $paper.title_raw }}",
+    authors: [
+      {{ range $paper.author }}
+      {
+        first: "{{ .first }}",
+        last: "{{ .last }}",
+        {{ if .id }}
+        // optional author id key, used for disambiguation
+        id: "{{ .id }}",
+        {{ end }}
```

Comment:
Could it have unintended side effects if this field is stored, but non-visible? I can imagine e.g. someone editing author order by just overwriting the text boxes (instead of dragging), in which case the ID would be attached to an entirely wrong person. Are we fine with having to check for these issues later?